Tests/test_imageops.py

```diff
@@ -94,6 +94,22 @@ def test_scale(self):
         newimg = ImageOps.scale(i, 0.5)
         self.assertEqual(newimg.size, (25, 25))
 
+    def test_colorize(self):
+        # Test the colorizing function
+
+        # Grab test image
+        i = hopper("L").resize((15, 16))
+
+        # Test original 2-color functionality
+        ImageOps.colorize(i, 'green', 'red')
```

Comment:
It would be good to `assertEqual` some values from the returned image. For example, check the colour of a pixel on the left, on the right, and in the centre.

And/or `assert_image_equal` against a pre-colourised image.

Reply:
ok sounds good, working on that now